src/PensionsRegulatorApi/Application/Queries/GetValidatedOrganisationsHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using MediatR;
+using PensionsRegulatorApi.Data;
+using PensionsRegulatorApi.Domain;
+
+namespace PensionsRegulatorApi.Application.Queries
+{
+    public class GetValidatedOrganisationsHandler : RequestHandler<GetValidatedOrganisations, IEnumerable<Organisation>>
+    {
+        private readonly IOrganisationRepository _repository;
+
+        public GetValidatedOrganisationsHandler(IOrganisationRepository repository)
+        {
+            _repository = repository ?? throw new ArgumentNullException(nameof(repository));
```

Comment:
Pet hate...the null check will never ever be hit as the DI container won't be able to instantiate the class.